curvefs/src/metaserver/partition.cpp

```diff
@@ -220,10 +220,6 @@ MetaStatusCode Partition::GetInode(uint32_t fsId, uint64_t inodeId,
         return MetaStatusCode::PARTITION_ID_MISSMATCH;
     }
 
-    if (GetStatus() == PartitionStatus::DELETING) {
```

Comment:
Why a under-deleting partition is readable?

Reply:
When delete inode, the partition will get the inode first.